packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2538,7 +2539,10 @@ export type DeclarativeCallAgent = CallAgent & IncomingCallManagement;
 export type DeclarativeIncomingCall = IncomingCall;
 
 // @public
-export type DeclarativeTeamsCallAgent = TeamsCallAgent & IncomingCallManagement;
+export type DeclarativeTeamsCallAgent = TeamsCallAgent & TeamsIncomingCallManagement;
+
+// @beta
```

Comment:
I believe export DeclarativeIncomingCall is an error, for DeclarativeTeamsIncomingCall - I don't think we need it, since it is just a complete alias of TeamIncomingCall